core/src/main/java/org/elasticsearch/cluster/routing/UnassignedInfo.java

```diff
@@ -124,7 +126,13 @@
      * @param message more information about cause.
      **/
     public UnassignedInfo(Reason reason, String message) {
-        this(reason, message, null, reason == Reason.ALLOCATION_FAILED ? 1 : 0, System.nanoTime(), System.currentTimeMillis(), false);
+        this(reason,
+             message,
+             null,
+             reason == Reason.ALLOCATION_FAILED ? 1 : 0,
+             System.nanoTime(),
+             System.currentTimeMillis(),
+             false);
```

Comment:
please stop this reformatting madness 😞. So much of this PR is just reformatting existing code. This makes reviewing terribly annoying, especially as the PR stands already at 1000 lines of changed code.